tslearn/barycenters/dba.py

```diff
@@ -210,15 +212,23 @@ def _mm_assignment(X, barycenter, weights, metric_params=None):
     n = X.shape[0]
     cost = 0.
     list_p_k = []
-    for i in range(n):
-        path, dist_i = dtw_path(barycenter, X[i], **metric_params)
-        cost += dist_i ** 2 * weights[i]
-        list_p_k.append(path)
+    #with parallel_backend('threading'):
+    res = Parallel(backend='threading',prefer="threads",require='sharedmem',n_jobs=n_treads,verbose=10)(delayed(dtw_path)(barycenter, X[i], global_constraint="sakoe_chiba",sakoe_chiba_radius=1) for i in range(n))
+    paths, dists = zip(*res)
+    paths = list(paths)
+    dists = list(dists)
+    cost = numpy.sum(numpy.multiply(numpy.power(dists, 2),weights))
     cost /= weights.sum()
+    list_p_k = paths
+    #for i in range(n):
+    #    path, dist_i = dtw_path(barycenter, X[i], **metric_params)
+    #    cost += dist_i ** 2 * weights[i]
+    #    list_p_k.append(path)
+    #cost /= weights.sum()
     return list_p_k, cost
 
 
-def _subgradient_valence_warping(list_p_k, barycenter_size, weights):
+def _subgradient_valence_warping(list_p_k, barycenter_size, weights, n_treads):
```

Comment:
We use `n_jobs` everywhere as a parameter name, and `None` as its default, we should stick to the same convention here, I think.